RedditSharp/Things/Post.cs

```diff
@@ -225,14 +225,19 @@ public Task<Subreddit> GetSubredditAsync()
         /// </summary>
         /// <param name="limit">Maximum number of comments to return</param>
         /// <returns></returns>
-        public async Task<List<Comment>> GetCommentsAsync(int limit = 0)
+        public async Task<List<Comment>> GetCommentsAsync(int limit = 0, CommentSort sort = CommentSort.Best)
         {
             var url = string.Format(GetCommentsUrl, Id);
+
+            //Only 'best' comment sorting isn't named the same
+            if (sort == CommentSort.Best)
+                url = $"{url}?sort=confidence";
+            else
+                url = $"{url}?sort={sort.ToString().ToLower()}";
+
             if (limit > 0)
-            {
-                var query = "limit=" + limit;
-                url = string.Format("{0}?{1}", url, query);
-            }
```

Comment:
Can you add braces back in. I don't like non curly braced IF statements and they have screwed us once or twice in this wrapper before